src/Components/AddTime/AddTime.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import Select from 'react-select';
+import ReactModal from 'react-modal';
+import { CountdownCircleTimer } from 'react-countdown-circle-timer';
+import { NavLink } from 'react-router-dom';
+import extendedTimeOptions from '../../assets/extendedTimeOptions';
+import addTimeModalStyles from './jsxStyles/addTimeModalStyles';
+import addTimeDropdownStyles from './jsxStyles/dropdownStyles';
+
+ReactModal.setAppElement('#root');
+
+function AddTime( { setExtension, setEtaSeconds, modalIsOpen, closeModal } ) {
```

Comment:
This looks awesome. We'll have to brainstorm how we wait to hook up the end trip / extend the trip option for the message.